scarb/tests/build.rs

```diff
@@ -687,3 +687,49 @@ fn edition_must_exist() {
                  unknown variant `2021`, expected one of `2023_01`, `2023_10`, `2023_11`
         "#});
 }
+
+#[test]
+fn dev_dep_used_outside_tests() {
+    let t = TempDir::new().unwrap();
+    let q = t.child("q");
+    ProjectBuilder::start()
+        .name("q")
+        .build(&q);
+    ProjectBuilder::start()
+        .name("x")
+        .dev_dep("q", Dep.path("./q"))
+        .src("src/foo.cairo", r#"
+        use q::f0;
```

Comment:
This will break, the number `0` is non-deterministic and will be different if you run the full test suite. You have to provide library source code manually. ;) Reason? To enforce test developers to not rely on default values assumed by `ProjectBuilder`.